stripe-crypto-onramp/src/main/java/com/stripe/android/crypto/onramp/model/KycInfo.kt

```diff
@@ -0,0 +1,17 @@
+package com.stripe.android.crypto.onramp.model
+
+import androidx.annotation.RestrictTo
+import com.stripe.android.paymentsheet.PaymentSheet
+import kotlinx.serialization.SerialName
+import kotlinx.serialization.Serializable
+
+@Serializable
+@RestrictTo(RestrictTo.Scope.LIBRARY_GROUP)
+internal data class KycInfo(
```

Comment:
Can we use the complete model in the [API review](https://docs.google.com/document/d/1fiRBeegAqMbaR7OUDUy0EA4idvHEDh_FcBE2DTz2vyo/edit?tab=t.0#heading=h.570u4leviueu)?

<img width="1151" height="273" alt="Image" src="https://github.com/user-attachments/assets/748fbee6-1089-48a3-a700-a689953dad5c" />

Reply:
for the model, this maps what the crypto endpoint accepts so we should be good!